cromulent/model.py

```diff
@@ -627,6 +628,9 @@ def _set_magic_resource(self, which, value, inversed=False):
 		if not current:
 			object.__setattr__(self, which, value)
 		elif type(current) is list:
+			# check value not in list
+			if self._factory.multiple_instances_per_property == "error" and isinstance(value, BaseResource) and value in current:
+				raise DataError("Cannot add the same resource in the same property more than once")
```

Comment:
Could the error message here include a mention of the property to change if this isn't the approach you want? Something like

    Change `model.factory.multiple_instances_per_property` to `"error"` or `"drop"` to change this behavior.

Reply:
:+1: